docs/configuration/remote-admin.mdx

```diff
@@ -115,6 +115,7 @@ Legacy admin is enabled using the Legacy Admin channel option in [Security Confi
    meshtastic --set security.admin_key "PASTEPUBLICKEYHERE"
```

Comment:
```suggestion
   meshtastic --set security.admin_key "base64:PASTEPUBLICKEYHERE"
```

Reply:
would this change be enough to clarify that 'base64' needs to be present?

Reply:
Possibly, and I'll leave it at your discretion. I considered that originally, but if you do retrieve your key via the CLI, it will already have the `base64:` prepended onto the key, and obviously you don't want to do something like `base64:base64:PASTEPUBLICKEYHERE` as a result.

Hopefully folks would be able to intuit that and so it would be a moot point, but...frankly I don't know. Thoughts?

Reply:
I'd like to think the average user is smart enough to solve that one.  Personally I find less words = more better.

Reply:
And applied!